src/main/webapp/app/exercises/shared/result/result.utils.ts

```diff
@@ -103,6 +118,22 @@ export const getUnreferencedFeedback = (feedbacks: Feedback[] | undefined): Feed
     return feedbacks ? feedbacks.filter((feedbackElement) => !feedbackElement.reference && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED) : undefined;
 };
 
```

Comment:
Generally, might it make sense to name all „AI“ related flags something with Athena as there might be additional, AI but not Athena related usecases in the future?